src/test/java/picard/util/LiftoverVcfTest.java

```diff
@@ -1169,4 +1173,49 @@ public void testLiftOverNoCallAndSymbolic(final LiftOver liftOver, final Variant
             Assert.assertEquals(resultAlleles, result.getAttributeAsStringList(LiftoverVcf.ORIGINAL_ALLELES, null));
         }
     }
+
+    @Test()
+    public void testNoDictionary() throws IOException {
+        final Path liftOutput = Files.createTempFile("tmpouput", ".vcf");
+        liftOutput.toFile().deleteOnExit();
+        final Path rejectOutput = Files.createTempFile("tmpreject", ".vcf");
+        rejectOutput.toFile().deleteOnExit();
+        final Path input = TEST_DATA_PATH.toPath().resolve("testLiftoverBiallelicIndels.vcf");
+        final Path referenceCopy = Files.createTempFile("ref", ".fasta");
+        referenceCopy.toFile().deleteOnExit();
+        Files.copy(REFERENCE_FILE.toPath(), referenceCopy, StandardCopyOption.REPLACE_EXISTING);
+        final String[] args = new String[]{
+                "INPUT=" + input.toString(),
+                "OUTPUT=" + liftOutput.toString(),
+                "REJECT=" + rejectOutput.toString(),
+                "CHAIN=" + CHAIN_FILE,
+                "REFERENCE_SEQUENCE=" + referenceCopy.toString(),
+        };
+        Assert.assertEquals(runPicardCommandLine(args), 1);
+    }
+
+    @Test(expectedExceptions = SAMException.class)
```

Comment:
is this the most specific exception that you can catch?

Reply:
As far as class, yes, this actually gets thrown out of the _AbstractFastaSequenceFile_ constructor, calling _IOUtil.assertFileIsReadable_, which throws a _SAMException_.  but I can add an _expectedExceptionMessageRegExp_ to the test.